app/views/users/show.html.haml

```diff
@@ -0,0 +1,39 @@
+%section.user.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      %article
+        %header
+          %img(src="http://www.gravatar.com/avatar/#{Digest::MD5.hexdigest(@user.email.downcase)}?s=40")
+          %h2= "#{@user.full_name}'s video collections (#{@user.queue_items.count})"
```

Comment:
This does not have to be all Ruby code.

`%h2 #{@user.full_name}'s video collections (#{@user.queue_items.count})`

should work as well.